aptos-move/framework/aptos-framework/sources/dispatchable_fungible_asset.move

```diff
@@ -72,6 +75,8 @@ module aptos_framework::dispatchable_fungible_asset {
                 features::dispatchable_fungible_asset_enabled(),
                 error::aborted(ENOT_ACTIVATED)
             );
+            fungible_asset::assert_not_frozen(store);
```

Comment:
Can you leave IF ELSE style comments on the corresponding function in fungible_asset and here to make sure any checks added in the future are present in both to avoid similar cases from happening?

Reply:
why not make it a friend function in `fungible_asset.move` called `sanity_check` or sth to reuse code and avoid omission.